geometry/latlon.cpp

```diff
@@ -2,28 +2,30 @@
 
 #include <sstream>
 
-using namespace std;
-
 namespace ms
 {
 // static
-double const LatLon::kMinLat = -90;
-double const LatLon::kMaxLat = 90;
-double const LatLon::kMinLon = -180;
-double const LatLon::kMaxLon = 180;
+double const LatLon::kMinLat = -90.0;
+double const LatLon::kMaxLat = 90.0;
+double const LatLon::kMinLon = -180.0;
+double const LatLon::kMaxLon = 180.0;
 
-string DebugPrint(LatLon const & t)
-{
-  ostringstream out;
-  out.precision(20);
-  out << "ms::LatLon(" << t.lat << ", " << t.lon << ")";
-  return out.str();
-}
+// Note. LatLon(-180.0, -180.0) are an invalid coordinates which are used in statistics.
```

Comment:
s/an invalid/invalid/
or
is a pair of invalid coordinates

s/which/that/

Reply:
Fixed.